api/spec/v1.yaml

```diff
@@ -42,6 +42,14 @@ x-query-params:
     description: |
       The block height from which to query state. The Oasis Indexer does not
       make any guarantees about availability of historical state data.
+  - &runtime
+    in: path
+    name: runtime
+    required: true
+    schema:
+      $ref: '#/components/schemas/Runtime'
+    description: |
+      The layer for which to return the transaction volume timeline.
```

Comment:
```suggestion
      The layer for which to return the requested data timeline.
```

Reply:
Uh bad copy paste. Rewrote from scratch. Thank you!